tests/tmt/tests/jats/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -eux
+
+TESTS_RESULT=test_results
+JAT_LOG="/var/tmp/jat/last/log.yaml"
+JAT_RELICS="/var/tmp/jat/last/relics/xcase-results.yaml"
+
+suite=${TESTSUITE:-leapp}
+testname=${TESTNAME:-cli}
+
+mkdir "$TESTS_RESULT-$suite" && cd "$TESTS_RESULT-$suite" && echo "Preparing to run tests for $suite.."
+for jats_test in $(find /usr/share/jats/suite/$suite/$testname -name test | sed -e 's/test$//'); do
+    echo "Processing $jats_test.."
+    test_dir_name=$(echo "$jats_test" | sed -e 's/\//_/g')
+    mkdir $test_dir_name
+    # save overall test result
```

Comment:
My gut tells me tmt should have built-in capabilities to log and report stuff nicely, but I haven't got to that yet